src/Utilities/Microphysics/Microphysics.jl

```diff
@@ -0,0 +1,115 @@
+"""
+    Microphysics
+
+Microphysics parameterization functions (for example condensation,
+evaporation, autoconversion rates for cloud water and rain water)
+TODO: Snow, ice, and 2-moment formulation
+"""
+module Microphysics
+
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+
+using Printf
+
+# rates of conversion between microphysics categories
+export qv2qli, ql2qr, qr2qv
```

Comment:
It would be good to stay with the naming conventions we had established, and improve readability. E.g., ``q_v_to_q_liq``, ``q_liq_to_q_rain`` etc.

Reply:
Can we discuss those conventions at some point?  The shum is not a standard way to mark specific humidities. Also the function names and variable names will continue to get longer if we stay this explicit.

Reply:
Longer is good if it improves readability. We do not need to use ``shum`` but can use ``q`` as in other parts of the code and the design documents. Basically, the closer we stay to the standard notation in the design docs, the easier it will be for users.